components/ide/jetbrains/image/BUILD.js

```diff
@@ -0,0 +1,85 @@
+const ideConfigs = [
```

Comment:
Do we need copyright here?

Reply:
Updated, see [diff](https://github.com/gitpod-io/gitpod/compare/5bdb2742e569ce8f22b842caa220b00863e65a2a..76d795de32ac1e1a01f350492e6a97cc1aa64a09)

Going to unhold this PR